Gemfile

```diff
@@ -49,6 +51,8 @@ group :development do
 end
 
 group :test do
+  gem 'pry-byebug'
+  gem 'pry-rails'
```

Comment:
I don't think these gems are used in the `test` environment. Also it causes a warning in bundler:

```
Your Gemfile lists the gem pry-byebug (>= 0) more than once.
You should probably keep only one of them.
While it's not a problem now, it could cause errors if you change the version of just one of them later.
```